sources/_sector/dex/trades/apechain/_sources.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+
+sources:
+  - name: saru_apechain
+    tables:
+      - name: sarupair_evt_swap
+      - name: sarufactory_evt_paircreated
```

Comment:
saru has 0 rows because of this source is empty:
https://github.com/duneanalytics/spellbook/actions/runs/13524413073/job/37791248642?pr=7656#step:12:23

Reply:
But the contract has been submitted and decoded.

Reply:
i'm assuming the wrong ABI was used then. it should be a simple uni v2 fork.